### PR TITLE
Dev/log video

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**
Added `log_video` function, that logs video file -> playable from artifacts tab.

### CODE REVIEW

File: neptunecontrib/api/video.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    _exp = experiment if experiment else neptune
+
+    name, file_ext = os.path.split(path_to_file)[1].split('.')
+
+    if video_name is None:
+        video_name = name
+    else:
+        assert isinstance(video_name, str), 'video_name must be string, got {}'.format(type(video_name))
+
+    encoded_sound = base64.b64encode(open(path_to_file, 'rb').read())
```
Comment (szymon-kuklewicz) [inline neptunecontrib/api/video.py:72]: encoded_video ?

**Review by szymon-kuklewicz — CHANGES_REQUESTED** [review]

File: neptunecontrib/api/video.py (hunk 1 above)
Comment (kamil-kaczmarek) [reply neptunecontrib/api/video.py:72]: yes, absolutely, let me change it

Comment (kamil-kaczmarek) [inline neptunecontrib/api/video.py:72]:
```suggestion
    encoded_video = base64.b64encode(open(path_to_file, 'rb').read())
```

File: neptunecontrib/api/video.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    html = """<!DOCTYPE html>
+        <html>
+        <body>
+
+        <video controls>
+          <source src='data:video/{};base64,{}'>
+        </video>
+
+        </body>
+        </html>""".format(file_ext, encoded_sound.decode())
```
Comment (kamil-kaczmarek) [inline neptunecontrib/api/video.py:82]:
```suggestion
        </html>""".format(file_ext, encoded_video.decode())
```

**Review by szymon-kuklewicz — APPROVED** [review]

**Pull request merged**